ep005/README.md

```diff
@@ -0,0 +1,58 @@
+# [the python @property decorator (beginner - intermediate)](https://youtu.be/orp6bhe4i00)
+
+In this video I explain how python implements getters and setters as well as an example where you might use them!  As a bonus I also show @cached_property
```

Comment:
probably want to put this in backticks so it doesn't @-mention github.com/cached_property

Reply:
It looks like it doesn't do it, but ok: https://github.com/int3l/explains/blob/e4a95a45f3bacb3a01d7e777f5982c457a45258f/ep005/README.md

Reply:
Maybe if someone create account with this nickname, it will start working 👍